enterprise/server/raft/registry/registry.go

```diff
@@ -305,6 +314,8 @@ func (d *DynamicNodeRegistry) pushUpdate(req *rfpb.RegistryPushRequest) {
 	}
 }
 
+// queryPeers looks up and adds one peer of the specified node to the registry
+// including the node info (shard ID and replica ID), NHID, and addresses.
```

Comment:
I might say something like:

// queryPeers queries the gossip network for node that hold the specified shardID and replicaID.
// If any nodes are found, they are added to the static registry.